blueprint/api.apib

```diff
@@ -2624,6 +2597,33 @@ This endpoint allows you to check whether a username is valid (by running a vali
 + id: `1` (string, required)
 + type: `stripe-customer`
 
+## Stripe Plan Attributes (object)
++ amount: 999 (number) - Describes the amount of a plan in cents.
++ created: `2016-07-08T03:03:51.967Z` (string)
```

Comment:
I would write that this is the `created` time from Stripe. Did we do this in #383?